src/SFA.DAS.CommitmentPayments.WebJob/Updater/DataLockUpdater.cs

```diff
@@ -133,6 +123,21 @@ public class DataLockUpdater : IDataLockUpdater
             }
         }
 
+        private async Task AutoResolveDataLockIfApprenticeshipStoppedAndBackdated(DataLockStatus datalock)
+        {
+            var apprenticeship = await
+                _apprenticeshipRepository.GetApprenticeship(datalock.ApprenticeshipId);
+
+            if (apprenticeship.PaymentStatus == PaymentStatus.Withdrawn &&
+                apprenticeship.StopDate == apprenticeship.StartDate)
+            {
+
+                _logger.Info($"Auto-resolving datalock for Apprenticeship withdrawn effective at start date {datalock.ApprenticeshipId} " +
```

Comment:
This log line would be easier to read and less likely to be misinterpreted, if the interpolation expression for AppenticeshipId was moved to just after the word Apprenticeship, so instead of...
"Auto-resolving datalock for Apprenticeship withdrawn effective at start date 301218..."
it read...
"Auto-resolving datalock for Apprenticeship #301218 withdrawn effective at start date ..."